src/Service/SchemaGenerator.php

```diff
@@ -77,9 +82,12 @@ protected function getAvailableMethods(array $exclude = [])
         // Get all methods enabled on the site
         $allMethods = MethodRegistry::singleton()->getMethods();
 
+        // Get the backup method as we omit this from the "available" methods.
+        $backupMethod = Config::inst()->get(MethodRegistry::class, 'default_backup_method');
+
         // Compile details for methods that aren't already registered to the user
         foreach ($allMethods as $method) {
-            if (in_array($method->getURLSegment(), $exclude)) {
+            if (in_array($method->getURLSegment(), $exclude) || get_class($method) === $backupMethod) {
```

Comment:
Same thing here with the `is_a()` instead to handle subclasses. Perhaps a utility function like `->isDefaultBackupMethod($method)` somewhere?

Reply:
Yep I've added a utility method to `MethodRegistry`.